samples/cpp/benchmark_app/main.cpp

```diff
@@ -606,6 +606,10 @@ int main(int argc, char* argv[]) {
                 device_nstreams.erase(device);
             }
         }
+        for (auto&& item : config) {
+            core.set_property(item.first, item.second);
```

Comment:
It breaks virtual plugins. `ov::device::properties` are disallowed in `set_property`

Reply:
Thanks for the review!
Sorry, didn't get the idea. Could you please describe it more broadly?
Breaks virtual plugins how?

Reply:
> Thanks for the review! Sorry, didn't get the idea. Could you please describe it more broadly? Breaks virtual plugins how?

Such plugins as AUTO/MUTLI/HETERO allows to set device properties via `ov::device::properties`. You mentioned commit from PR https://github.com/openvinotoolkit/openvino/pull/15175 which disallows setting such properties via core. They must be set in config of compile_model. Are you sure that problem is not in VPUX?

Reply:
> Are you sure that problem is not in VPUX?

Pretty much yes. Because the only change I made when I started CI was to update the OV commit to a recent commit on the master branch. And I got the results that are shown in the PR description.

Okay, I will see what I can do here.
Thanks!

Reply:
Discussed this PR offline with @yangwang201911. 
Thanks @yangwang201911!
I will modify this PR shortly.

Reply:
Okay, it turned out to be very simple :)
The problem only occurred when running a benchmark_app that used .blob files.
```
./benchmark_app -m model.blob ...
```
This is due to the fact that when switching to `device_config`, the `import_model` call remained unchanged, without the `device_config` parameter.

Reply:
The CI check is now in progress. But the fix worked locally.